trino/client.py

```diff
@@ -127,6 +131,7 @@ def __init__(
         self._roles = roles.copy() if roles is not None else {}
         self._prepared_statements: Dict[str, str] = {}
         self._object_lock = threading.Lock()
+        self._timezone = timezone or get_localzone_name()
```

Comment:
is there a way to validate the time zone using tzlocal?

Reply:
Good point. I added the check with ZoneInfo